packages/liquidator/src/constants.js

```diff
@@ -0,0 +1,15 @@
+// As defined in 1inch
+const ALTERNATIVE_ETH_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
+
+const ONE_SPLIT_ADDRESS = "0xC586BeF4a0992C495Cf22e1aeEE4E446CECDee0E";
+
+const GAS_LIMIT_BUFFER = 1.25;
```

Comment:
I really like having a constants file specifically for `liquidator`, @chrismaree we should import this from the liquidator to use for the `createLiquidation` and `withdrawRewards` functions